src/details/ArborX_Callbacks.hpp

```diff
@@ -170,7 +170,7 @@ KOKKOS_INLINE_FUNCTION
   return false;
 }
 
-template <typename Callback, typename Predicates>
+template <typename Value, typename Callback, typename Predicates>
 void check_valid_callback(Callback const &callback, Predicates const &)
```

Comment:
Is there a good reason not to pass the values as an argument and just deduce `Value` from it?

Reply:
I haven't thought this through. Do we require that values are default constructable?

Reply:
I realize now we don't have the `values` argument to the acceleration data structure constructor when we validate the callback.  Please ignore.